projects/igniteui-angular/src/lib/directives/toggle/toggle.directive.ts

```diff
@@ -182,7 +182,9 @@ export class IgxToggleDirective implements IToggleView, OnInit, OnDestroy {
             return;
         }
 
-        this._overlayId = this.overlayService.attach(this.elementRef, overlaySettings);
+        if (!this._overlayId) {
```

Comment:
Saw the ID gets reset on close, but since we use the `info` above - can this check use it instead?
Just in case the id becomes invalid somehow.

Reply:
much better in this way. Thank you!